openrouteservice/src/main/java/org/heigit/ors/api/requests/matrix/MatrixRequest.java

```diff
@@ -101,6 +103,12 @@
     @JsonIgnore
     private boolean hasOptimized = false;
 
+    @ApiModelProperty(name = PARAM_OPTIONS,
+            value = "For advanced options formatted as json object. For structure refer to the [these examples](https://GIScience.github.io/openrouteservice/documentation/routing-options/Examples.html).",
+            example = "{\"avoid_borders\":\"controlled\"}")
```

Comment:
I believe`@ApiModelProperty` should include `hidden = true` unless we want to expose advanced matrix `options` parameter to the public API.

Reply:
We do want to expose them. They are the benefit of using core matrix.

Reply:
Sure, makes sense. For this, however, https://github.com/GIScience/openrouteservice/pull/959#discussion_r714031788 needs to be adapted. As soon as `options` parameter to matrix requests is exposed in the public API a way of handling it efficiently is necessary.

Reply:
Is it still usable when hidden? What does hidden do?

Reply:
It's usable but does not show up in the interactive docs.

Reply:
done